app/build.gradle

```diff
@@ -153,8 +153,12 @@ dependencies {
     androidTestImplementation('com.schibsted.spain:barista:3.6.0') {
         exclude group: 'org.jetbrains.kotlin'
     }
-    androidTestImplementation "org.mockito:mockito-android:3.4.6"
-    androidTestImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0'
+    androidTestImplementation "org.mockito:mockito-android:3.5.10"
+    androidTestImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0") {
```

Comment:
I didn't want to add the brackets for consistency with the rest of the file but it seems like they're necessary when using curly braces. Maybe I used the wrong syntax. Either way, I'm happy to lose the brackets if there's a way to do it.